antarest/login/web.py

```diff
@@ -172,11 +172,17 @@ def groups_get_all(
 
     @bp.get("/groups/{id}", tags=["User"])
     def groups_get_id(
-        id: str, current_user: JWTUser = Depends(auth.get_current_user)
+        id: str,
+        verbose: Optional[int] = None,
```

Comment:
this is not "verbose", `list_user` or `details` could be better and with boolean type